core/trino-main/src/test/java/io/trino/server/security/oauth2/TestingHydraIdentityProvider.java

```diff
@@ -334,7 +364,7 @@ public static void main(String[] args)
                                     .put("http-server.authentication.oauth2.client-secret", "trino-secret")
                                     .put("http-server.authentication.oauth2.audience", "https://localhost:8443/ui")
                                     .put("http-server.authentication.oauth2.user-mapping.pattern", "(.*)@.*")
-                                    .put("oauth2-jwk.http-client.trust-store-path", Resources.getResource("cert/localhost.pem").getPath())
+                                    .put("oauth2-jwk.http-client.trust-store-path", service.getCertPath() + "/localhost.pem")
```

Comment:
`certDir` + "/localhost.pem" + "/localhost.pem"?
Run this class locally.

Reply:
Whops, I missed this, thanks! Fixed and tested locally.